src/averageAges.js

```diff
@@ -55,10 +82,30 @@ function calculateWomenAverageAge(people, withChildren) {
  * @return {number}
  */
 function calculateAverageAgeDiff(people, onlyWithSon) {
-  // 1. find a mother of each person (or only for men)
-  // 2. keep people who have mothers in the array
-  // 3. calculate the difference child.born - mother.born
-  // 4. return the average value
+  const MALE_SIGN = 'm';
+
```

Comment:
You use this const few times in your code. Mabe, it make sense to put it at the very beggining of the script and just reuse?